maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -186,6 +190,13 @@ private DistroProperties resolveDistroProperties(Server server) throws MojoExecu
 					}
 					platformMode = false;
 					break;
+				case O3_REFERENCE_APPLICATION:
+					distroProperties = DistroHelper.getDistroPropertiesFromFile(new File(SDKConstants.MAVEN_PLUGINS_RESOURCE_PATH + "openmrs-distro-3.0.properties"));
+					File source = new File(SDKConstants.MAVEN_PLUGINS_RESOURCE_PATH + "configuration");
+					FileUtils.copyDirectoryToDirectory(source, server.getServerDirectory());
+					FileUtils.copyToDirectory(new File(SDKConstants.MAVEN_PLUGINS_RESOURCE_PATH + "omod/initializer-2.5.2.omod"), new File(server.getServerDirectory(), SDKConstants.OPENMRS_SERVER_MODULES));
```

Comment:
why do you hard code the version of the initialiser module.  Set up a regex that copies the `.omod` for  ⬆️ 2.5 of the initialiser module.